plugins/smart_proxy_dynflow/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-smart-proxy-dynflow (0.8.11-1) stable; urgency=low
```

Comment:
Right, I'm a bit slower today :/ It should be better now. Thank for spotting this.